source/common/quic/platform/quiche_flags_impl.h

```diff
@@ -11,12 +11,25 @@
 
 #include "absl/container/flat_hash_map.h"
 
+// Not wired into command-line parsing.
+#define DEFINE_QUIC_COMMAND_LINE_FLAG_IMPL(type, flag, value, help)                                \
+  quiche::TypedFlag<type>* FLAGS_##flag = new TypedFlag<type>(#flag, value, help);
+
 namespace quiche {
 
 const std::string EnvoyQuicheReloadableFlagPrefix =
     "envoy.reloadable_features.FLAGS_quic_reloadable_flag_";
 const std::string EnvoyFeaturePrefix = "envoy.reloadable_features.";
 
+// TODO(mpwarres): implement. Lower priority since only used by QUIC command-line tools.
+inline std::vector<std::string> quicParseCommandLineFlagsImpl(const char* /*usage*/, int /*argc*/,
```

Comment:
The name reflects `quic` instead of `quiche`, does it make sense to move around this function?